### PR TITLE
Update documentation 2.0

### PR DESCRIPTION
**Pull request by timkimadobe**
## Description

This PR updates:
1. API docs link and function signature
2. New getting started test app guide (should be aligned with [Edge Android repo's version](https://github.com/adobe/aepsdk-edge-android/blob/main/Documentation/getting-started-test-app.md))
3. Update getting started guide (should be aligned with [Edge Android repo's version](https://github.com/adobe/aepsdk-edge-android/blob/main/Documentation/getting-started.md))
4. Main README 
    * Aligned with [Edge README](https://github.com/adobe/aepsdk-edge-android)
    * Update related resources table 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: Documentation/getting-started-test-app.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Getting started with the test app
+
+## Data Collection mobile property prerequisites
+
+The test app needs to be configured with a Data Collection mobile property with the following extensions before it can be used:
+
+* [Mobile Core](https://github.com/adobe/aepsdk-core-android) (installed by default)
+* Edge Network (this extension)
```
Comment (kevinlind) [inline Documentation/getting-started-test-app.md:8]: This is the Edge Bridge repo. Can you include a link to the Edge Network repo?

File: Documentation/getting-started-test-app.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+* [Identity for Edge Network](https://github.com/adobe/aepsdk-edgeidentity-android)
+* [Assurance](https://github.com/adobe/aepsdk-assurance-android)
+
+See [Configure the Edge Network extension in Data Collection UI](getting-started.md#configure-the-edge-network-extension-in-data-collection-ui) for instructions on setting up a mobile property.
```
Comment (kevinlind) [inline Documentation/getting-started-test-app.md:12]: Update the link to point to the Edge Network Android extension repo.

File: Documentation/getting-started-test-app.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
+# Getting started with the test app
+
+## Data Collection mobile property prerequisites
+
+The test app needs to be configured with a Data Collection mobile property with the following extensions before it can be used:
```
Comment (kevinlind) [inline Documentation/getting-started-test-app.md:5]: Can you add a line here, or a note, stating that Edge Bridge does not contain a card/extension in Data Collection? You can use the same note in `getting-started.md` line 21.

**Review by kevinlind — COMMENTED** [review]
In `getting-started.md` under Install the extension, I remember discussing removing the line which sets the log level from the examples as it's optional and in production we don't recommend setting the level to verbose. Can you remove those lines from the Java and Kotln examples?
`MobileCore.setLogLevel(LoggingMode.VERBOSE)`

**Review by addb — APPROVED** [review]

**Review by kevinlind — APPROVED** [review]

**Pull request merged**